src/apps/common/submodules/portListing/views/ports-listing.html

```diff
@@ -47,11 +53,10 @@
 					</a>
 				</td>
 			{{/if}}
-				{{!-- <td data-filter-value="{{state}}{{#if extra.isScheduledToday}} scheduledtoday{{/if}}" data-sort-value="{{ tryI18n @root.i18n.portListing.misc.status state }}{{#compare state '===' 'scheduled'}}-{{scheduled_date}}{{/compare}}"> --}}
 				<td data-filter-value="{{state}}" data-sort-value="{{ tryI18n @root.i18n.portListing.misc.status state }}">
 				{{#compare state '===' 'submitted'}}
 					<div class="status-wrapper">
-						<i class="icon-telicon-prioritize"></i>
+						{{telicon "list-prioritize"}}
 						<div class="content">
 							<div class="status">
 								{{@root.i18n.portListing.misc.status.submitted}}
```

Comment:
Here you could use `{{tryI18n @root.i18n.portListing.misc.status status}}`, and for the other statuses as well, to not to depend on the `{{#compare...`, as a first step to remove repeated tags. I wonder if something similar could be done for the icons. If not, then the `{{#compare...` could be kept only to render the icons, instead of each content div.

Reply:
Thanks, but i'm not sure if this is really useful on this case